packages/utilities/psammead-styles/src/colours.js

```diff
@@ -6,7 +6,7 @@ export const C_WHITE = '#FFFFFF';
 export const C_BLUEJAY = '#0F556C';
 export const C_BLUEJAY_LHT = '#C3DEE7';
 export const C_OAT_LHT = '#F5F3F1';
-export const C_PEBBLE = '#5C5752';
+export const C_PEBBLE = '#AEAEB5';
```

Comment:
Have we confirmed with UX that this is fine across the board? I dont think this is used anywhere, but in theory it would affect in all places

Reply:
I've confirmed that they don't want this old Pebble, and I've checked the psammead and simorgh repos and neither are using `C_PEBBLE`. I don't know anywhere else importing colours from psammead-styles